src/scripts/main.js

```diff
@@ -1,3 +1,39 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('li>ul>li');
+const topHeaders = document.querySelectorAll('ul>li');
+
+function spanWrap(input) {
```

Comment:
The function name 'spanWrap' is not descriptive. Consider using a more descriptive function name like 'wrapContentInSpan'.